cli_reference/developer_cli_odo/understanding-odo.adoc

```diff
@@ -121,3 +121,4 @@ wildfly     openshift     10.0,10.1,11.0,12.0,13.0,14.0,15.0,16.0,17.0,18.0,19.0
 ----
 +
 The `TAGS` column represents the available image versions, for example, `10` represents the `rhoar-nodejs/nodejs-10` container image.
+To learn more about the CLI commands, go to xref:../../cli_reference/openshift_developer_cli/odo-cli-reference.adoc#basic-odo-cli-commands_odo-cli-reference[odo CLI reference].
```

Comment:
Apologies for the delay! I tinkered with the xref a bit and got it to work as follows: 

If you want to learn more about the CLI commands, see xref:../../cli_reference/developer_cli_odo/odo-cli-reference.adoc#basic-odo-cli-commands_odo-cli-reference[odo CLI reference].

Reply:
@bmcelvee, no worries about the "delay." Funnily enough, I was working on sending up a new commit while you were reviewing it. I noticed the same thing. 

I updated the XREF, and when I locally build it, it works for me. I also shortened the wording a little bit. 

Also, noted about the merge freeze. As long as it's merged next week, we should be fine.

Reply:
Comments addressed, so I've resolved the conversation.